composer.json

```diff
@@ -21,11 +21,10 @@
         "composer-plugin-api": "^1"
     },
     "require-dev": {
-        "php": ">=7",
         "composer/composer": "dev-master",
-        "eloquent/phony-phpunit": "^3",
+        "eloquent/phony-phpunit": "^1",
         "errors/exceptions": "^0.2",
-        "phpunit/phpunit": "^6"
+        "phpunit/phpunit": "^5"
```

Comment:
outch :)

Reply:
yes otherwise even when running `composer update --no-dev` it was failing because that version of phpunit doesn't support php5